lldb/source/Plugins/Process/Utility/NativeProcessSoftwareSingleStep.cpp

```diff
@@ -94,6 +94,39 @@ static lldb::addr_t ReadFlags(NativeRegisterContext &regsiter_context) {
                                                  LLDB_INVALID_ADDRESS);
 }
 
+static int GetSoftwareWatchpointSize(const ArchSpec &arch,
```

Comment:
Doesn't this return the size of the *break*point (i.e. it should be called `GetSoftwareBreakpointSize` ?

Reply:
Fixed, changed to `GetSoftwareBreakpointSize`